src/server/controllers/match.controller.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+
+const matchSchema = require('mongoose').model('match');
+import mailer from 'nodemailer';
+
+/*
+ * Methods about mentoring request, accept or reject including E-mail Service
+ */
+
+const EMAIL_SUBJECT = 'New mentee needs your help!';
+const EMAIL_HTML = '<h1>Hi,</br> new mentee needs your mentoring.</h1>';
+const YODA_ACCOUNT = '"Yoda Service Team" <yoda.mentor.lab@gmail.com>';
+
+let matchData = new matchSchema();
+let sampleFailResult = {
+  successCode: 0,
+  errPoint: null,
+  err: null,
+};
+
+// Send mentoring request pushing Email to mentor(receiver)
+function sendRequestEmail(res, mentor, mentee, content) {
+  let transport
+    = mailer.createTransport('smtps://yoda.mentor.lab%40gmail.com:svbootcamp@!@smtp.gmail.com');
+  let mailOptions = {
+    from: YODA_ACCOUNT,
+    to: mentor,
+    replyTo: mentee,
+    subject: EMAIL_SUBJECT,
+    html: EMAIL_HTML + content,
```

Comment:
Add a TODO to properly format the email body. Right now, we add an HTML header which may be followed by just a string text.